clients/fides-js/src/fides.ts

```diff
@@ -223,6 +225,13 @@ const _Fides: FidesGlobal = {
       // Nothing to show if there's no experience
       return false;
     }
+    if (
+      this.options.fidesKnownPreference === ConsentMethod.ACCEPT ||
+      this.options.fidesKnownPreference === ConsentMethod.REJECT
+    ) {
+      // If consent preference was automatic, we should not show the experience
```

Comment:
Note to re-word this based on our `fides_consent_override` naming decision: `If consent preference override exists, we should not show the experience`